ppb/mouse.py

```diff
@@ -0,0 +1,53 @@
+from typing import Callable
+
+from ppb.vector import Vector
+
+
+class Mouse:
+    """
+    Object interface to the hardware mouse.
+
+    """
+    def __init__(self, press_function: Callable=None,
+                 move_function: Callable=None, move_to_function: Callable=None):
+        self.position = Vector(0, 0)
+        self.screen_position = Vector(0, 0)
+        self.buttons = [False, False, False]
+
+        self.press_function = press_function or _not_implemented
+        self.move_function = move_function or _not_implemented
+        self.move_to_function = move_to_function or _not_implemented
```

Comment:
if you did `partial(_not_implemented, "...")` with the error messages you use below you could avoid the wrapper methods and just assign to `press()` directly here

Reply:
You mean as the second half of the assignment here?

If so, I can dig it, I'll throw that together and see how it works.